recipes/doxygen/all/conanfile.py

```diff
@@ -47,23 +49,26 @@ def configure(self):
                 tools.Version(self.version) == "1.8.18"):
             raise ConanInvalidConfiguration("Doxygen version {} broken with VS {}.".format(self.version,
                                                                                            self.settings.compiler.version))
-        del self.settings.compiler.cppstd
```

Comment:
Don't we need to add a `configure` method that removed `compiler.libcxx` and `compiler.cppstd`?

Reply:
Not sure. If we are already removing `self.info.settings.compiler` in the `package_id` method, CCI will compute duplicated values and build only one binary... and it might be useful for someone to use the actual value of these subsettings if they are building from sources. wdyt?

Reply:
The package id will be the same, but the build process will be different.
When you keep `compiler.cppstd`, the build helpers will insert `-std=c++XX` from the profile (if set).

Perhaps not removing `compiler.cppstd` is more correct.
doxygen contains .cpp sources.

So let's keep you changed it to.